open_seq2seq/data/text2speech/text2speech.py

```diff
@@ -0,0 +1,290 @@
+# Copyright (c) 2018 NVIDIA Corporation
+
+from __future__ import absolute_import, division, print_function
+from __future__ import unicode_literals
+from six.moves import range
+
+import string
+import os
+import six
+import numpy as np
+import tensorflow as tf
+import pandas as pd
+
+from open_seq2seq.data.data_layer import DataLayer
+from .speech_utils import get_speech_features_from_file, get_mel, inverse_mel, normalize, denormalize
+from open_seq2seq.data.utils import load_pre_existing_vocabulary
+
+
+class Text2SpeechDataLayer(DataLayer):
+  """Text-to-speech data layer class, that uses ``tf.data`` API.
```

Comment:
This is no longer applicable since there is no other API than tf.data anymore :)

Reply:
This should be fixed.